CONTRIBUTING.md

```diff
@@ -79,13 +79,16 @@ Helpful issues usually include:
 
 Contributions to the documentation site are made by editing the docs repository. You can do this directly on GitHub.com or by creating a copy of the repository locally, making your changes there, and contributing back to our repository.
 
+> By default, your merged PR to an English page will trigger our Translation Status Tracker. If your change should **NOT** be applied to every language (e.g. a typo fix to an English word), please include the word "minor" in your PR title and/or in a commit message.  See the next section for more details.
```

Comment:
I think only the squash merge commit (that generally is the title) is used to mark the PR as minor/major. If this is really the case, we need to change the wording a bit to not imply that you can select which pages changed are considered minor/major changes in the same PR.

Reply:
Ah, I hadn't considered people might think that only certain commits would trigger. 

So maybe, although it's really coming from commit message, I'll just be sneaky here and say PR title for simplicity.